sdcm/mgmt/cli.py

```diff
@@ -953,7 +953,7 @@ def get_cluster_hosts_with_ips(db_cluster):
         return [[n, n.ip_address] for n in db_cluster.nodes]
 
     def add_cluster(self, name, host=None, db_cluster=None, client_encrypt=None, disable_automatic_repair=True,  # pylint: disable=too-many-arguments
-                    auth_token=None):
+                    auth_token=None, credentials=None):
```

Comment:
Please, describe this new parameter in the docstring below.

Reply:
Added it and other missing parameters